code/modules/robotics/robot/robot_docking_station.dm

```diff
@@ -974,7 +1071,7 @@ TYPEINFO(/obj/machinery/recharge_station)
 		if("cell-install")
 			if (!isrobot(src.occupant))
 				return
-			if (user == src.occupant)
+			if (user == src.occupant && !isdrone(user))
```

Comment:
Does this mean drones *can* switch out their own cell? 🤔

Reply:
Huh. No idea why that's there, no. Might have been leftover from before I realized AI eyes being able to change cells was a bug? Either way.